Makefile

```diff
@@ -183,12 +183,24 @@ docker-build: test
 # Build docker image with buildx
 # Experimental docker feature to build cross platform multi-architecture docker images
 # https://docs.docker.com/buildx/working-with-buildx/
-docker-buildx:
+docker-buildx-dev:
 	export DOCKER_CLI_EXPERIMENTAL=enabled
 	@if ! docker buildx ls | grep -q container-builder; then\
 		docker buildx create --platform "linux/amd64,linux/arm64,linux/arm/v7" --name container-builder;\
 	fi
-	docker buildx build -t ${IMG} --platform "linux/amd64,linux/arm64,linux/arm/v7" . --push
+	docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" \
+		-t $(REPOSITORY):dev \
+		. --push
+
+docker-buildx-release:
+	export DOCKER_CLI_EXPERIMENTAL=enabled
+	@if ! docker buildx ls | grep -q container-builder; then\
+		docker buildx create --platform "linux/amd64,linux/arm64,linux/arm/v7" --name container-builder;\
```

Comment:
do you need to add `--use` here to it would use `container-builder` context?

Reply:
That is correct, the only other alternative would be to run something like `docker buildx use container-builder` although I like your suggestion better - forgot to add it.